lib/xboxrt/libc_extensions/stdlib_ext_.h

```diff
@@ -0,0 +1,12 @@
+#ifdef __cplusplus
```

Comment:
How would you like to license this code?

Reply:
MIT license note added.